<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateMainMnemonic</class>
 <widget class="QDialog" name="UpdateMainMnemonic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update Additional Mnemonic</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(101, 101, 101);</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="9" column="1">
    <widget class="QLabel" name="label_input_info">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox_sensor_name">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(217, 217, 217);</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_5">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(101, 101, 101);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>sensor_name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(101, 101, 101);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>main_mnemonic</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_main_mnemonic_name">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(217, 217, 217);</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(101, 101, 101);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>decription</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_main_mnemonic_description">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(217, 217, 217);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(101, 101, 101);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>type_of_unit</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="comboBox_type_of_unit">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(217, 217, 217);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_main_mnemonic_description">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(101, 101, 101);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>unit</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="comboBox_unit">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(217, 217, 217);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="pushButton_CLEAN">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(217, 217, 217);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>CLEAN</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="pushButton_INPUT">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(217, 217, 217);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>INPUT</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
